kernel-open/nvidia/nv-acpi.c

```diff
@@ -424,7 +424,7 @@ static void nv_acpi_event(acpi_handle handle, u32 event_type, void *data)
 
     if (event_type == NVIF_DISPLAY_DEVICE_CHANGE_EVENT)
     {
-        /* We are getting NVIF events on this machine. We arent putting a very
+        /* We are getting NVIF events on this machine. We aren't putting a very
```

Comment:
:(

Reply:
Could you elaborate on the your issue with this change?